quarkus/runtime/pom.xml

```diff
@@ -100,6 +100,15 @@
             <artifactId>quarkus-logging-json</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.apicatalog</groupId>
+            <artifactId>titanium-json-ld</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.setl</groupId>
+            <artifactId>rdf-urdna</artifactId>
+        </dependency>
+
```

Comment:
I have reviewed added dependencies by this PR.
It is OK to merge the PR as we are able to create productized versions of them.

Reply:
@pskopek Thank you a lot! @mposolda any other issues that I should solve before we can merge the PR?